DangoPlop/Assets/Scripts/Ball_Spawn.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Ball_Spawn : MonoBehaviour {
```

Comment:
let's call this Ball_Factory instead. In the future this class needs an array that keeps track of all of the balls spawned in the playing field. This will let us do any function with the balls very quickly, instead of finding all the balls first via tag and then doing the balls' function. no need to create the array now. just change the classname